cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1398,8 +1398,7 @@ def _add_head_node(self):
                     "touch": {"command": "touch /etc/chef/ohai/hints/ec2.json"},
                     "jq": {
                         "command": (
-                            "jq --argfile f1 /tmp/dna.json --argfile f2 /tmp/extra.json -n '$f1 * $f2' "
-                            "> /etc/chef/dna.json "
+                            'jq -s ".[0] * .[1]" /tmp/dna.json /tmp/extra.json > /etc/chef/dna.json '
```

Comment:
How is this change related to the support for AL2023?

Reply:
New versions of jq deprecated `--argfile`